code/datums/hud/critter.dm

```diff
@@ -375,7 +375,7 @@
 
 /// updates bleeding hud element
 /datum/hud/critter/proc/update_blood_indicator()
-	if (!src.bleeding) return //doesn't have a hud element to update
+	if (!src.bleeding || src.master.can_bleed) return //doesn't have a hud element to update
```

Comment:
not sure if you really need this second bit. It may even be the case that a mob still has a `bleeding` hud element to clear from a previous state of `can_bleed` true